package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "build": "remix vite:build",
     "dev": "remix vite:dev",
+    "watch-local": "tsx tools/watch-local.ts",
```

Comment:
We'd probably want to make so this automatically runs on vite dev but doesn't do anything if you're not in local mode.